regulations/static/regulations/css/less/typography.less

```diff
@@ -42,35 +42,50 @@ h3,
 h4,
 h5,
 h6 {
-    .font-bold;
-    margin-bottom: 0;
+    .sans-font-bold;
+    margin: 0.5em 0;
```

Comment:
Looks like we're switching to px; mind switching this here? Also, this is setting a bold font, but then swapping the font weight below. Should we just be setting the "regular" font?